src/python/CRABInterface/RESTUserWorkflow.py

```diff
@@ -278,7 +277,7 @@ def _checkSite(self, site, pnn = False):
         sites = self.allPNNNames.sites if pnn else self.allCMSNames.sites
         if site not in sites:
             excasync = ValueError("A site name you specified is not valid")
-            invalidp = InvalidParameter("The parameter %s is not in the list of known CMS sites %s" % (site, sites), errobj = excasync)
+            invalidp = InvalidParameter("The parameter %s is not in the list of known CMS PhEDEx nodes." % (site), errobj = excasync)
```

Comment:
Maybe we can put a url where people can find valid PNNs? Is there a link in phedex @belforte ?